pkg/scheduler/factory/plugins_test.go

```diff
@@ -34,14 +34,18 @@ func TestAlgorithmNameValidation(t *testing.T) {
 		"Some,Alg:orithm",
 	}
 	for _, name := range algorithmNamesShouldValidate {
-		if !validName.MatchString(name) {
-			t.Errorf("%v should be a valid algorithm name but is not valid.", name)
-		}
+		t.Run(name, func(t *testing.T) {
+			if !validName.MatchString(name) {
+				t.Errorf("should be a valid algorithm name but is not valid.")
```

Comment:
Since the subtest replaces some special chars with underscore, I wonder if we should keep the old format string here. WDYT?